### ISSUE TITLE
Could not assign default channel!

### ISSUE THREAD

**Issue by xRedFox** [TeaSpeak/TeaSpeak#254]
**Describe the bug**
Default channels invasion xD
```
<17:47:51> Trying to resolve hostname drc.thebunny.eu
<17:47:53> Trying to connect to server on drc.thebunny.eu:10004
<17:47:53> Could not assign default channel!
```

**To Reproduce**
![video 05-08-2018 17-46-32](https://user-images.githubusercontent.com/19707386/43687672-5a9d6366-98d9-11e8-9286-ba651275adf9.gif)

**Expected behaviour**
You know :)

**Logs**
[log_2018-08-05_14_49_33_general.log](https://github.com/TeaSpeak/TeaSpeak/files/2260458/log_2018-08-05_14_49_33_general.log)

**Environment information:**
Operating system: ArchLinux
TeaSpeak version: TeaSpeak-1.2.1-beta-1

**Comment by WolverinDEV** [comment]
Hey,
i cant see the streps to reproduce that.

**Comment by xRedFox** [comment]
Yeah that video wasn't made by me... Tomorrow I will give you a better one
